api/src/backend/tasks/jobs/scan.py

```diff
@@ -237,6 +237,14 @@ def perform_prowler_scan(
 
                     status = FindingStatus[finding.status]
                     delta = _create_finding_delta(last_status, status)
+                    first_seen = datetime.now(tz=timezone.utc)
+                    # When the delta attribute is different from new it means that previously there was a finding with the same uid and delta new, so we will obtain the first_seen attribute of that first finding
+                    if delta != Finding.DeltaChoices.NEW:
```

Comment:
If the backfill is not included this should be changed to always get the previous finding `first_seen` if the `delta != new`.